packages/ast-spec/src/legacy-fixtures/basics/fixtures/type-import-type-with-import-attributes/fixture.ts

```diff
@@ -0,0 +1,3 @@
+// TODO: This fixture might be too large, and if so should be split up.
+
+type A = import("A", { with: { type: "json" } });
```

Comment:
Is it correct to have the fixture in `packages/ast-spec/src/legacy-fixtures`, or should this be in a different directory?

Reply:
Different directory: `packages/ast-spect/src/` -> something other than `legacy-fixtures`.

Example PR: #9972